### DISCUSSION TITLE
Examples of data transformation

### DISCUSSION THREAD

**Discussion by asherccohen** [crimx/observable-hooks#51] [Q&A]
It would be interesting to see some official examples of how data transformation can be achieved with this library.

The examples could show how data from a fetch can be converted (maybe normalized) or how a series of transformations can benefit of observables.

**Comment by crimx** [comment]
Hi. This library is about marrying RxJS and React. There are [examples](https://observable-hooks.js.org/examples/) on the docs on how to achive that. It assumes you know about both libraries.

observable-hooks allows you to do basically whatever you want with observables. So just check out other tutorials out there if you want to learn about observable patterns.
